lib/Autocomplete.js

```diff
@@ -103,18 +103,9 @@ let Autocomplete = React.createClass({
     if (this.keyDownHandlers[event.key])
       this.keyDownHandlers[event.key].call(this, event)
     else {
-      const { selectionStart, value } = event.target
-      if (value === this.props.value)
-        // Nothing changed, no need to do anything. This also prevents
-        // our workaround below from nuking user-made selections
-        return
       this.setState({
         highlightedIndex: null,
         isOpen: true
-      }, () => {
-        // Restore caret position before autocompletion process
-        // to work around a setSelectionRange bug in IE (#80)
-        this.refs.input.selectionStart = selectionStart
```

Comment:
This line of code without the fix above causes user-made selection (i.e. shift+arrow) to be completely broken. And this, to reiterate, was the fix for the incredibly weird typeahead bug we had in IE. Without this fix `Autocomplete` is unusable in IE.